tool/release.sh

```diff
@@ -56,7 +56,13 @@ git checkout -- package.json
 git checkout -- CHANGELOG.md
 npm run changelog
 VERSION_NUM="$(node -p "require('./package.json').version")";
-echo "version number for the build is" $VERSION_NUM
+echo "recommended version number for the build is" $VERSION_NUM 
+
+read -p "do you want to continue with the recommended version number? [y/n] " yn
```

Comment:
seems like the default is `y` but not mentioned.

Reply:
ahh, you can ignore this, seems like the rest of the file also uses that pattern